src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorTableValue.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Entities;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    // Returned by AddTabularConnector
+    // Doesn't contain any ServiceProvider which is runtime only
+    public class ConnectorTableValue : TableValue, IRefreshable
+    {
+        public new TableType Type => _tabularService.TableType;
```

Comment:
>public new TableType Type => _tabularService.TableType; [](http://example.com/codeflow?start=8&length=55)

Do we need this? 
Doesn't TableValue.Type already have correct type?

Reply:
Absolutely required, yes!
I dont' remember why but I'll confirm tomorrow

Reply:
Not needed anymore